modules/systemd.nix

```diff
@@ -99,6 +99,11 @@ let
       ) + "\n";
     };
 
+  sd-switch =
+    let
+      url = "https://gitlab.com/rycee/sd-switch/-/archive/master/sd-switch-master.tar.bz2";
```

Comment:
Will this be pinned at some point (e.g. when this PR is no longer a draft/considered "ready")? Rubs me the wrong way to depend on a moving target like this.

Reply:
Yes, this PR can only be merged once this program has been in Nixpkgs for some time.

Reply:
Great, thanks for the clarification. I have at least one MR I'll be making to `sd-switch` in the coming days (or when I can find time) -- I love the choice to write it in Rust ;^)